contracts/test/LightClient.t.sol

```diff
@@ -21,7 +21,7 @@ contract LightClientCommonTest is Test {
     uint32 public constant BLOCKS_PER_EPOCH_TEST = 3;
     uint32 public constant DELAY_THRESHOLD = 6;
     uint32 public constant MAX_HISTORY_SECONDS = 1 days;
-    uint32 initialEpoch = 1 days;
+    uint32 public initialEpoch = 1 days;
```

Comment:
Why do we need to keep this variable?

Reply:
ah this epoch isn't used for testing the block per epoch functionality but for testing the retention period for storing the state history on the contract

Reply:
I see, should we change the name to avoid confusion?

Reply:
done here https://github.com/EspressoSystems/espresso-sequencer/commit/5acfd9472fc649eb6c8d6c76507789a318e55a99